fingertip/plugins/self_test/ssh_terminate.py

```diff
@@ -0,0 +1,31 @@
+# Licensed under GNU General Public License v3 or later, see COPYING.
+# Copyright (c) 2020 Red Hat, Inc., see CONTRIBUTORS.
+
+import fingertip
+
+
+def check_processes(m):
+    with m:
+        if hasattr(m, 'console'):
+            # There should not be any hanging process after installation
+            m.console.sendline("ps ax | grep '[s]shd.*notty")
```

Comment:
This ain't checking for anything besides the presence of the prompt. What you probably want is to inspect `.before`: https://pexpect.readthedocs.io/en/stable/overview.html

Reply:
Right. The point of this test is that there is no other output than the prompt from this command. After having a look in morning, I see above is missing matching `'` in the end, which made the console hang. With correctly matching apostrophes it works as expected.